platform/nativescript/util/index.js

```diff
@@ -1,6 +1,6 @@
-import { isKnownView, getViewMeta, getViewClass } from '../element-registry'
+import { isKnownView, getViewMeta, getViewClass } from '../element-register'
 import { makeMap, once } from 'shared/util'
-import { VUE_VM_REF } from '../runtime'
+ const VUE_VM_REF = '__vue_vm_ref__'
```

Comment:
@farfromrefug Why do we need to re-declare rather than importing? Is this due to the fact that `runtime/index` relies on `tns-core-modules` and should not be referenced to avoid build issues?

Reply:
@rigor789 yes indeed. May be we should have a ```constants.ts``` with the constant define ?

Reply:
@farfromrefug yes, I like the idea of a constants file - should keep the codebase tidier!

Reply:
done